Bebb-Protocol-and-Apps/PWS
# Create space from uploaded model

Adds a new section on the Create space view; users can now select a glb file from their device, see the model in a preview scene and then create a new space with the model file to be included (as seen in the preview).

Open: When loading the new space, the uploaded model file currently isn't loaded (via its URL, which hits the backend canister's HTTP file to return the file).

To test:
Make sure the local setup still works (see below)
go to the Create tab on the page and scroll to the upload model section
select a file
preview the scene with the model
confirm the space creation
see the confirmation that the space was created
see the new space under My Spaces
load the new space
the space should look like the preview (including the uploaded model) [see Open; this currently doesn't work as the model isn't being loaded via its URL]

Local setup changes:
this PR also includes changes to the local setup in response to upgrading dfx to the latest version
the local Bebb Protocol canister is receiving a different canister id now (which seems stable and is updated as part of the PR)
you might not need to upgrade dfx to run and test this
